File: src/app/todo/todo.ts
/**
 *
 * (c) 2013-2017 Wishtack
 *
 * $Id: $
 */

export class Todo {

    id?: string;
    description?: string;

    constructor(args: Todo) {
        this.id = args.id;
        this.description = args.description;
    }

}

File: backend/todo/todo.py
#-*- coding: utf-8 -*-
#
# (c) 2013-2017 Wishtack
#
# $Id: $
#

from synthetic import synthesize_constructor, synthesize_property


@synthesize_constructor()
@synthesize_property('id')
@synthesize_property('description')
class Todo(object):
    pass
